projects/storefrontlib/src/lib/ui/pages/404/404.component.html

```diff
@@ -1,20 +0,0 @@
-<div class="cx-page">
```

Comment:
Why do we remove this component implementation? Shouldn't it be just transformed to a CMS component?

Reply:
There is already a component in the backend. But fair point, I've created #1254 in order to have the same UI back (but then CMS driven).